src/drivers/imx/pmc-ipc.c

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
This file is probably not needed unless you need to perform IPC with an external PMIC.

Reply:
> This file is probably not needed unless you need to perform IPC with an external PMIC.

Indeed. Will fix in v2.